### PR TITLE
3 Books License Added

### PR DESCRIPTION
**Pull request by Aryan9inja**
## What does this PR do?
Add info on basis of #11355 

### Description
Add License Notes to following Books:

1. [Eloquent JavaScript 3rd edition](http://eloquentjavascript.net) - Marijn Haverbeke (HTML, PDF, EPUB, MOBI)

### License Line Found Under The Book Image On Site
"Licensed under a [Creative Commons attribution-noncommercial license](https://creativecommons.org/licenses/by-nc/3.0/)."

2. [Introduction to Scientific Programming with Python](https://library.oapen.org/bitstream/id/56d27e73-e92a-4398-8198-239be7aacc93/2020_Book_IntroductionToScientificProgra.pdf) - Joakim Sundnes (PDF)

### License Line Found On Page 5 Of PDF
"This book is licensed under the terms of the Creative Commons Attribution 4.0
International License (http://creativecommons.org/licenses/by/4.0/)"

3. [Think Python 2nd Edition](https://greenteapress.com/wp/think-python-2e/) - Allen B. Downey (3.x) (HTML, PDF)

### License Line Found On Page 4 Of PDF
"Permission is granted to copy, distribute, and/or modify this document under the terms of the
Creative Commons Attribution-NonCommercial 3.0 Unported License, which is available at http:
//creativecommons.org/licenses/by-nc/3.0/"

## Checklist:
- [  ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [  ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [  ] Include author(s) and platform where appropriate.
- [  ] Put lists in alphabetical order, correct spacing.
- [  ] Add needed indications (PDF, access notes, under construction).
- [  ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1142,7 +1142,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Designing Scalable JavaScript Applications](https://www.manning.com/books/designing-scalable-javascript-applications) - Emmit Scott (PDF+livebook)
 * [Dev Docs](https://devdocs.io/javascript/) - Various (HTML)
 * [DOM Enlightenment](https://frontendmasters.com/guides/javascript-enlightenment/) - Cody Linley
-* [Eloquent JavaScript 3rd edition](http://eloquentjavascript.net) - Marijn Haverbeke (HTML, PDF, EPUB, MOBI)
+* [Eloquent JavaScript 3rd edition](http://eloquentjavascript.net) - Marijn Haverbeke (HTML, PDF, EPUB, MOBI) (CC BY-NC-ND)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1145]: wrong license

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1992,7 +1992,7 @@ Books on general-purpose programming that don't focus on a specific language are
     * [Introduction to Programming Using Python](http://python-ebook.blogspot.co.uk) - Cody Jackson (1st edition) (2.3)
 * [Introduction to Python](http://kracekumar.com/post/71171551647/introduction-to-python) - Kracekumar (2.7.3)
 * [Introduction to Python for Econometrics, Statistics and Numerical Analysis](https://www.kevinsheppard.com/files/teaching/python/notes/python_introduction_2020.pdf) - Kevin Sheppard (PDF) (3.8)
-* [Introduction to Scientific Programming with Python](https://library.oapen.org/bitstream/id/56d27e73-e92a-4398-8198-239be7aacc93/2020_Book_IntroductionToScientificProgra.pdf) - Joakim Sundnes (PDF)
+* [Introduction to Scientific Programming with Python](https://library.oapen.org/bitstream/id/56d27e73-e92a-4398-8198-239be7aacc93/2020_Book_IntroductionToScientificProgra.pdf) - Joakim Sundnes (PDF) (CC BY-NC-ND)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1995]: wrong license

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -2072,7 +2072,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [The Standard Python Library](https://web.archive.org/web/20200626001242/http://effbot.org/librarybook/) - Fredrik Lundh *(:card_file_box: archived)*
 * [Think Complexity](https://greenteapress.com/wp/think-complexity-2e/) - Allen B. Downey (2nd Edition) (PDF, HTML)
 * [Think DSP - Digital Signal Processing in Python](https://greenteapress.com/wp/think-dsp/) - Allen B. Downey (PDF, HTML)
-* [Think Python 2nd Edition](https://greenteapress.com/wp/think-python-2e/) - Allen B. Downey (3.x) (HTML, PDF)
+* [Think Python 2nd Edition](https://greenteapress.com/wp/think-python-2e/) - Allen B. Downey (3.x) (HTML, PDF) (CC BY-NC-ND)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:2075]: wrong license

**Review by eshellman — CHANGES_REQUESTED** [review]
use the specified licenses!

**Comment by Aryan9inja** [comment]
Sir I have added changes.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**